app/views/users/choose_identity.html.erb

```diff
@@ -7,7 +7,7 @@
 	<%- if @current_user.is_a?(PubcookieUser) -%>
 		<li><%= link_to "Dream Project mentor", update_identity_path(:identity => "Mentor"), :method => :post %></li>
 	<%- else -%>
-		<li><%= link_to "high-school student", update_identity_path(:identity => "Student"), :method => :post %></li>
+		<li><%= link_to "high-school student", update_identity_path(:identity => "Participant"), :method => :post %></li>
```

Comment:
Yeah. When you're messaging this functionality, you'll want to make sure to emphasize the need to use a login link, otherwise mentees might get confused and log in on their own and click this link, which associated their account to a compeltely different person.